### ISSUE TITLE
Select URLs are unable to be added as curated results

### ISSUE THREAD

**Issue by ghost** [mwmbl/mwmbl#195]
Some URLs are unable to be curated to the search results for unknown reasons. While I've ran into this issue perhaps three times now, the example I can recall off hand is with ReverbNation, where attempting to add the https version of the site's homepage [here](https://www.reverbnation.com/) simply doesn't work.

Will add other such examples if needed upon coming across them.

**Comment by daoudclarke** [comment]
Yup, ReverbNation times out on our request, perhaps it doesn't like our user agent. We need to return an error message in this case.

**Comment by sk33z3r** [comment]
I seem to be having issues with any URL I attempt to add to curated results, whether I enter it with https or http.

1. Make a query
2. Rank some results
3. Don't see the site I expect, click `Add +`
4. Enter the URL
5. Click `Save`

From here nothing occurs. I see HTTP 500 in the logs for the `/app/add` endpoint. Because of #214 , there isn't an obvious reason why it failed.

This is the request that was responded to with a 500. Looks pretty much identical to all the requests I've tried, other than the URL for `new_url`

```
XHRPOST
https://mwmbl.org/app/add/
[HTTP/2 500  1723ms]

	
csrfmiddlewaretoken	"randomtoken"
url	[…]
title	[…]
extract	[…]
score	[…]
state	[…]
new_url	"https://wphp.org"
query	"physician health program"
curation_id	"8383"
```

I've attempted with these URLs for this query (both http and https):
* `www.fsphp.org`
* `gaphp.org`
* `wphp.org`
* `cphp.org`

If I go to the home page and use https://mwmbl.org/app/domain-submissions/new, it added the first one I tried without error.
